BUILD.gn

```diff
@@ -376,25 +376,46 @@ template("create_dist_template") {
   }
 }
 
-create_dist_template("create_native_symbols_dist") {
-  output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_arch-native-symbols.7z"
-  dir_inputs = [ "$brave_product_name.pdb.syms" ]
-  deps = [ "app/$current_os:symbol_dist_resources" ]
-}
+if (should_generate_breakpad_symbols) {
+  create_dist_template("create_breakpad_symbols_dist") {
+    if (is_android) {
+      output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_android_base-$target_cpu-symbols-$target_android_output_format.zip"
+    } else {
+      output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_arch-symbols.zip"
+    }
 
-create_dist_template("create_symbols_dist") {
-  if (is_android) {
-    output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_android_base-$target_cpu-symbols-$target_android_output_format.zip"
-  } else {
-    output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_arch-symbols.zip"
+    deps = [ "//brave/app/$current_os:symbol_dist_resources" ]
+
+    dir_inputs = [ "$brave_product_name.breakpad.syms" ]
   }
 
-  deps = [ "app/$current_os:symbol_dist_resources" ]
-  if (brave_debug_symbol_level == 2 && is_win) {
```

Comment:
The idea of `brave_debug_symbol_level` was use 1 for breakpad only and 2 for breakpad + native.

Now `brave_debug_symbol_level == 1` seems to be unused => Let's remove `brave_debug_symbol_level` by replacing to `should_generate_breakpad_symbols` (maybe rename to `should_generate_symbols`?)

Reply:
yep, I've noticed `brave_debug_symbol_level == 1` wasn't used anywhere. Removed it at all and renamed to `should_generate_symbols`.